### ISSUE TITLE
Cell properties shows style elements that are not allowed through allowedContent

### ISSUE THREAD

**Issue by ideb** [ckeditor/ckeditor-dev#1986]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

This is a followup to a previous issue https://github.com/ckeditor/ckeditor-dev/issues/1014

Which is based on an issue in Drupal: https://www.drupal.org/project/drupal/issues/2785483

1. Assume the following allowedContent: 
```
config.allowedContent = {
		td: {
			attributes: 'colspan rowspan',
			styles: false
		}
	}
```
2. Add a table and open the 'Cell properties' dialog

### Expected result

* The 'Cell properties' dialog shows form elements for 'Cell Type', 'Rows Span', 'Columns Span'

### Actual result

* The 'Cell properties' dialog still shows form elements for 'Word Wrap', 'Horizontal Alignment', 'Vertical Alignment', 'Background Color', 'Border Color'

## Other details

* Browser: All
* OS: All
* CKEditor version: -
* Installed CKEditor plugins: tabletools

**Comment by engineering-this** [comment]
I'm confirming that you can add cell styling via dialog, that are disallowed. When going to `source mode` and back they will be removed.

**Comment by Comandeer** [comment]
Merged in https://github.com/ckeditor/ckeditor-dev/commit/3280058ea989725e292bfdbed5c7962c5dd9f116.

**Comment by mlewand** [comment]
This issue has been reverted in #2735 as it caused a nasty regression #2732 which was a release blocker.

**Comment by Comandeer** [comment]
Merged into 4.11.3.

**Comment by mlewand** [comment]
@engineering-this @Comandeer please have a look at built version, guys are reporting that build version is failing on table-related tests. I have a strong feeling that this is result of the merge.

**Comment by Comandeer** [comment]
@mlewand, the issue was connected with the method for getting appropriate menu item in test. I've prepared PR with the fix → #2820.
